src/build.py

```diff
@@ -110,7 +110,12 @@ def run_step(returncode, msg):
                 CLONE_DIR_PATH,
                 dict(
                     headers=dict([('cache-control', cache_control)]),
-                    excludePaths=['*/Dockerfile', '*/docker-compose.yml'],
+                    excludePaths=[
+                        '*/Dockerfile',
+                        '*/docker-compose.yml',
+                        '*/federalist.json',
+                        '*/pages.json'
```

Comment:
[question] I wonder about this one just because it seems possible that folks may have a `pages.json` somewhere within their site's structure that is not related to cloud.gov Pages. Should these be scoped to the root?

Reply:
we can start with that